core/google/api/core/retry.py

```diff
@@ -146,3 +189,129 @@ def retry_target(target, predicate, sleep_generator, deadline):
         time.sleep(sleep)
 
     raise ValueError('Sleep generator stopped yielding sleep values.')
+
+
+@six.python_2_unicode_compatible
+class Retry(object):
+    """Exponential retry decorator.
+
+    This class is a decorator used to add exponential back-off retry behavior
+    to an RPC call.
+
+    Although the default behavior is to retry transient API errors, a
+    different predicate can be provided to retry other exceptions.
+
+    Args:
+        predicate (Callable[Exception]): A callable that should return ``True``
+            if the given exception is retryable.
+        initial (float): The minimum about of time to delay. This must
+            be greater than 0.
+        maximum (float): The maximum about of time to delay.
+        multiplier (float): The multiplier applied to the delay.
+        jitter (float): The maximum about of randomness to apply to the delay.
+        deadline (float): How long to keep retrying.
+    """
+    def __init__(
+            self,
+            predicate=if_transient_error,
+            initial=1,
+            maximum=60,
+            multiplier=2,
```

Comment:
Shouldn't you follow your own documented types here? (i.e. use `1.0`, `60.0` and `2.0`)